configs/components/fesom/fesom-2.1.yaml

```diff
@@ -38,6 +38,75 @@ choose_version:
+  2.1-recom-par_tracers: # MA: for Kai testing ESM-Tools in Levante
+    branch: feature/par_tracers
+    git-repository:
+    - https://gitlab.dkrz.de/FESOM/fesom2.git
+    destination: fesom-2.1-recom-par_tracers
+
+  2.1-recom-par_tracers_mpi: # kh 13.04.22 for testing FESOM-REcoM MPI based tracer loop parallelization with ESM-Tools in Levante
+    branch: awiesm2-recom-parallel
+    git-repository:
+    - https://gitlab.dkrz.de/FESOM/fesom2.git
+    destination: fesom-2.1-recom-par_tracers_mpi
+# kh 08.03.22 todo: merge conflict (if enabled)
+#   include_models:
+#       - recom
+# YY: 29.04.22 to compile wiso code
+    wiso_code: true
+
+# MA: changes specific to PAR-TRACERS go here so they don't destroy other working versions of FESOM-2.1)
+# TODO: remove this block when PAR-TRACERS is generalized!
+par_tracers: "$(( 'par_tracers' in  '${fesom.version}' ))"
+choose_fesom.par_tracers:
+    True:
+        # kh 21.02.22 use uppercase here
+        forcing_data_dir: "${pool_dir}/FORCING/"
+        # kh 21.02.22
+        asforcing: CORE2
+        namelists:
+                - namelist.config
+                - namelist.forcing
+                - namelist.oce
+                - namelist.ice
+                - namelist.io
+        # kh 21.02.22 temporary disabled to get fesom-2.1-recom-par_tracers running on levante
+        #       - namelist.cvmix
+        #       - namelist.icepack
+        # kh 14.04.22 last used for core2 resolution on levante
+        # kh 19.04.22 disable for haralds's tests
+        nproc: 1024
+        # kh 19.04.22 temporary disabled to get fesom-2.1-recom-par_tracers_mpi running on levante
+        remove_fesom.namelist_changes.namelist.forcing.nam_sbc:
+            - nm_xwind_file
+            - nm_ywind_file
+            - nm_humi_file
+            - nm_qsr_file
+            - nm_qlw_file
+            - nm_tair_file
+            - nm_prec_file
+            - nm_snow_file
+            - nm_mslp_file
+            - nm_runoff_file
+            - nm_sss_data_file
+        config_files:
+                config:  config
+                forcing: forcing
+                ice:     ice
+                oce:     oce
+                io:      io
+        # kh 21.02.22 temporary disabled to get fesom-2.1-recom-par_tracers running on levante
+        #       cvmix:   cvmix
+        #       icepack: icepack
+        # kh 21.02.22 these entries are required in combination with the corresponding entries in "config_sources:" to copy/use the defaults like namelist.recom, ...
+                recom:   recom
+                io.recom.ciso: io.recom.ciso
+        # kh 21.02.22 these entries are required in combination with the corresponding entriesin "config_files:" to copy/use the defaults like namelist.recom, ...
+        add_config_sources:
+                recom:   "${namelist_dir}/namelist.recom"
+                io.recom.ciso: "${namelist_dir}/namelist.io.recom.ciso"
+#-----------END OF the SPECIAL CHANGES FOR FESOM-PAR-TRACERS
```

Comment:
General idea: we should probably try to move towards putting comments and such in commit messages, and (maybe) not (or as well?) in the code. It makes it cluttered to read, in my opinion.

Reply:
After the problem of the namelist references in general way is solved, also the still existing comments in the yaml code become obsolete. Until then, I find it quite helpful - in a feature branch - to have them available directly in the yaml code (without the need to use git diff or similar). Partly the comments also reflect (after a previous analysis) how the ESM Tools "work". In principle, this can also be done through the documentation, but having it at hand in the exact applicable place is sometimes also helpful.

Reply:
Making the online documentation more clear was something we also discussed during our retreat. That comment was just here as a reminder to myself that we should not forget about it.

Reply:
I agree with both, in my opinion, as this block will disappear in the future, we can leave the comments there as they are useful to @khimstedt